### PR TITLE
Add basic Excel-oriented import-export example

### PR DESCRIPTION
**Pull request by arielsaffer**
Here's a basic version of the example notebooks demonstrating the Excel import and export options. I used the simple format (not the "user friendly" one). Let me know if there's anything that would be useful to add.

Uses Python API to run the simulation (not CLI).

The notebook is computed.

### CODE REVIEW

File: examples/notebooks/run_from_xlsx.ipynb [hunk 1]
```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# PoPS Border simulation using XLSX file upload\n",
```
Comment (wenzeslaus) [inline examples/notebooks/run_from_xlsx.ipynb:7]: XLSX file configuration? (there can be couple different XLSX files involved)

File: examples/notebooks/run_from_xlsx.ipynb [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+    "from popsborder.simulation import run_simulation\n",
+    "from popsborder.inputs import load_configuration\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Excel configuration should include all parameters related to:\n",
+    "1. [consignments](../../docs/consignments.md) (what is imorted, from where, in what amounts), \n",
```
Comment (wenzeslaus) [inline examples/notebooks/run_from_xlsx.ipynb:31]: imported

File: examples/notebooks/run_from_xlsx.ipynb [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+    "    pretty=\"boxes\",\n",
+    "    verbose=True\n",
+    "    )\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The summary of results can also be converted to a tabular format and exported to Excel, or used further in Python with the pandas library."
```
Comment (wenzeslaus) [inline examples/notebooks/run_from_xlsx.ipynb:103]: Consider adding headings. One e.g. here.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
I'm not sure if we need example of output file in the documentation. Maybe makes sense when the notebook is computed?

File: examples/notebooks/run_from_xlsx.ipynb [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Excel configuration should include all parameters related to:\n",
+    "1. [consignments](../../docs/consignments.md) (what is imorted, from where, in what amounts), \n",
+    "2. [contaminants](../../docs/contamination.md) (proportion and distribution of contaminated consignments, boxes, and items), and \n",
+    "3. [inspection](../../docs/inspections.md) (sampling strategies and rates)\n",
+    "\n",
+    "There is a help file (linked above) for each of these sections that goes into greater detail about how to define the parameters."
```
Comment (wenzeslaus) [inline examples/notebooks/run_from_xlsx.ipynb:35]: Maybe also include text before every code cell describing the step. The two following code cells can be one introduced in a separate paragraph here as _Now we load the configuration from a file:_ or something like that.

**Comment by arielsaffer** [comment]
I've tried to address all of your comments in this new version.

**Review by wenzeslaus — APPROVED** [review]

**Comment by wenzeslaus** [comment]
Thanks, looks great, merged.

**Pull request merged**